pkg/containerutil/containerutil.go

```diff
@@ -331,6 +336,16 @@ func Stop(ctx context.Context, container containerd.Container, timeout *time.Dur
 	if err != nil {
 		return err
 	}
+	ipc, err := ipcutil.DecodeIPCLabel(l[labels.IPC])
+	if err != nil {
+		return err
+	}
+	// defer umount
+	defer func() {
+		if err := ipcutil.CleanUp(ipc); err != nil {
+			log.G(ctx).Warnf("failed to clean up IPC  container %s: %s", container.ID(), err)
```

Comment:
The `clean up IPC  container` may change to `clean up IPC container`

Reply:
Thanks a lot. Applied.